apdfl18/CPlusPlus/Getting_Started.html

```diff
@@ -161,11 +161,11 @@ <h2>Licensing</h2>
                         <p>The Activation application can be run directly and you can enter an Activation Key.</p>
                         <dl>
                             <dt><h3 id="cpyinfo">Copyright Information</h3></dt>
-                            <dd>Portions © 1984-2023  Adobe, Incorporated; Portions © 2000-2023 Datalogics, Inc. All rights reserved.<br>Use of Datalogics software is subject to the applicable license agreement.</dd>
+                            <dd>Portions © 1984-2024  Adobe, Incorporated; Portions © 2000-2023 Datalogics, Inc. All rights reserved.<br>Use of Datalogics software is subject to the applicable license agreement.</dd>
```

Comment:
There's another copyright on this line that mentions 2023.

Reply:
fixed